css/index.css

```diff
@@ -143,4 +237,133 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+header {
+    display: flex;
+    width: 880px;
```

Comment:
to make this more responsive, I would hardcode an overall pixel width in the `body` - like so:

```
body {
max-width: 880px;
width: 100%;
font-family: 'Roboto';
etc...
}
```

then you can use percentages in your elements to make them more responsive. 

```
header {
display: flex;
width: 100%;
}
```